spec/features/donation_site_spec.rb

```diff
@@ -28,6 +28,15 @@
     expect(page.find('.alert')).to have_content "added"
   end
 
+  scenario "User creates a new donation site on index" do
+    visit url_prefix + '/donation_sites'
+    donation_site_traits = attributes_for(:donation_site)
+    fill_in "donation_site_name", with: donation_site_traits[:name]
+    fill_in "donation_site_address", with: donation_site_traits[:address]
+    click_button "Create"
+    expect(page.find('tbody tr')).to have_content(donation_site_traits[:name])
```

Comment:
This is good but I think you might want to use a more explicit and unique string for the "name" -- using the default factory name is too likely to generate a false-positive, were a record to be added implicitly by another factory.

The address is fine (since you're not testing against it) but for name, maybe something like "Name Added Dynamically" or something similar.